codegen/templates/vec.rs.tera

```diff
@@ -1609,6 +1609,21 @@ impl {{ self_t }} {
         self + ((rhs - self) * s)
     }
 
+    /// Moves towards `rhs` based on the value `d`.
```

Comment:
I think it would be worth wrapping this in `{% if dim < 4 %}` because I don't know that it would be useful on the Vec4.

Reply:
On second thoughts, leave this as is, I already have a bunch of other methods on Vec4 that are mathematically valid but probably not useful, so this won't be out of place at least.